docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Clustering/KMeansWithOptions.cs

```diff
@@ -36,21 +39,27 @@ public static void Example()
             // Train the model.
             var model = pipeline.Fit(trainingData);
 
-            // Create testing data. Use different random seed to make it different from training data.
-            var testData = mlContext.Data.LoadFromEnumerable(GenerateRandomDataPoints(500, seed: 123));
+            // Create testing data. Use different random seed to make it different
+            // from training data.
+            var testData = mlContext.Data.LoadFromEnumerable(
+                GenerateRandomDataPoints(500, seed: 123));
 
             // Run the model on test data set.
             var transformedTestData = model.Transform(testData);
 
             // Convert IDataView object to a list.
-            var predictions = mlContext.Data.CreateEnumerable<Prediction>(transformedTestData, reuseRowObject: false).ToList();
+            var predictions = mlContext.Data.CreateEnumerable<Prediction>(
+                transformedTestData, reuseRowObject: false).ToList();
 
-            // Print 5 predictions. Note that the label is only used as a comparison wiht the predicted label.
-            // It is not used during training.
+            // Print 5 predictions. Note that the label is only used as a comparison
+            // wiht the predicted label. It is not used during training.
```

Comment:
Maybe also mention that the label is used to calculate Normalized Mutual Information when the model is evaluated later on in the sample.